docs/installation.md

```diff
@@ -72,37 +72,48 @@ If you want to help develop Lychee, install the development dependencies by remo
 
-#### Directory Permissions
-After installing Lychee, you need to configure some permissions. Directories within the `storage` and the `bootstrap/cache` directories should be writable by your web server or Lychee will not run. Additionaly `public/uploads`, `public/dist` and `public/sym` need to be writable to upload files, customise the CSS and utilise [symbolic links](https://lycheeorg.github.io/docs/settings.html#symbolic-link) respectively.
+#### Application Settings
 
-#### Copy configuration file
-If you have not copied the `.env.example` file to a new file named `.env`, you should do that now.
+The main directory should contain a `.env` file.
+Normally, Composer will have copied this file for you while installing the dependencies.
+If the `.env` file does not exist, then you should copy `.env.example` to a new file named `.env` now.
+Note that the `.env` file should **not** be exposed online as it contains the encryption key used for cookies as well as database credentials.
 
-#### Application Key
-The next thing you should do after installing Lychee is set your application key to a random string. This is easily done by using the `php artisan key:generate` command.
+##### Application Key
 
-If you don't want to use this command and set the key yourself, this string should be 32 characters long and can be set in the `.env` environment file.
+The `.env` file contains the setting `APP_KEY` which holds your application key.
+Normally, Composer sets this key for you while installing the dependencies.
+If it is missing, Lychee will not work. Then, you can use the `php artisan key:generate` command to set the application key to a random value.
 
-**If the application key is not set, your user sessions and other encrypted data will not be secure!**
+##### Additional Configuration
 
-#### Additional Configuration
-Lychee needs almost no other configuration out of the box. You are free to get started developing! However, you may wish to review the `config/app.php` file and its documentation. It contains some options that aren't included from `.env` that you may wish to change according to your installation.
+The default `.env` file provides usable settings out of the box.
 
-You may also want to configure a few additional components of Lychee in `.env`, such as:
+You may want to configure a few additional components of Lychee, such as:
 
 - Cache
 - Database
 - Session
 
+They are documented in the `.env.example` file, which may have been copied into your `.env`. They are also listed on the [configuration](configuration.html) page.
+
+##### Advanced Configuration (Dangerous)
+
+Some advanced options cannot be configured through the `.env` file. If you really need to change them, look in the `config/` directory.
+
+**If you don't know what they're doing, do not change them.**
+
 ## Web Server Configuration
 
 ### General
+
+Configure your web server's root to be the `public` directory.
```

Comment:
We've been asked about this recently I believe -- could we make this sentence more... nuanced? Many won't want to or even be able to change the root, so we shouldn't make it sound like this is a hard requirement. How about saying something like:

"Configure your web server to make the `public` directory accessible via a URL. This could be the root of your web server (short and convenient) or, if you prefer, a subfolder such as `photos`".

Reply:
That could work. If we do it may be worth reiterating that the level above shouldn't be publicly accessible.

Reply:
I'm in strong favor of both comments  Repeating the warning is something we should really do. It was only recently when we had the issue that a user made the root of Lychee publicly accessible and the user tried to access Lychee via putting `public/` in the URL.

Btw: It just crosses my mind that we do not mention that the setting `APP_URL` must be set to the _external_ URL by which the `public/` folder can be reached. This should be part of the configuration of the `.env` file but it won't harm to repeat it here again.